client/src/app/pages/applications/components/bulk-copy-assessment-review-form/bulk-copy-assessment-review-form.tsx

```diff
@@ -104,7 +104,10 @@ export const BulkCopyAssessmentReviewForm: React.FC<
       selectOptions: dedupeFunction(
         applications
           .map((app) => app.businessService?.name)
-          .map((name) => ({ key: name, value: name }))
+          .map((name) => ({
+            key: `BulkCopyAssessmentReviewForm-${name}`,
```

Comment:
Wouldn't we need to do this for all options with keys in the category list? What about the 'tags' options [here](https://github.com/konveyor/tackle2-ui/pull/511/files#diff-0631a29b250364a7dd66110d8c5567ed72ef22f5091b180a8be69fa818799bbdR146)

Reply:
I thought about that but this seems to be the only case where we're stacking filters.